src/org/labkey/targetedms/TargetedMSModule.java

```diff
@@ -411,6 +413,15 @@ public boolean isExactMatch()
                     v.setFrame(WebPartView.FrameType.PORTAL);
                     return v;
                 }
+            },
+
+            new BaseWebPartFactory(TARGETED_MS_RUNS_WEBPART_NAME)
```

Comment:
This doesn't seem right - presumably this should be TARGETED_MS_INSTRUMENT_SUMMARY

Reply:
oops, corrected